android/src/main/java/com/gettipsi/stripe/StripeModule.java

```diff
@@ -373,4 +371,36 @@ public void onResult(@NonNull BooleanResult booleanResult) {
       }
     );
   }
+
+  private String exist(final ReadableMap map, final String key) {
+    if (map.hasKey(key)) {
+      return map.getString(key);
+    } else {
+      return null;
```

Comment:
@fAns1k I believe method that returns `null` should have explicit comment about that or return empty string.